observability/logging/cluster-logging-support.adoc

```diff
@@ -66,10 +78,5 @@ You can use the xref:../../support/gathering-cluster-data.adoc#gathering-cluster
 
 For prompt support, supply diagnostic information for both {product-title} and {logging}.
 
-[NOTE]
```

Comment:
Seems this would be a place to mention the logging must-gather which replaced the "logging-dump" script" or reference the must-gather section

Reply:
The section after covers must-gather.